lib/useful/append.js

```diff
@@ -0,0 +1,28 @@
+import Promise from 'bluebird';
+import sentry from '../config/sentry';
+
+function append(bot, msg, suffix) {
+  if (!suffix) {
+    bot.sendMessage(msg.channel, 'Usage: **`!!`** `text`');
+    return;
+  }
+
+  Promise.resolve(msg.channel.messages.getAll('author', msg.author))
+    .then(messages => {
+      messages.pop();
+      if (messages[0] === undefined) {
+        return;
+      }
+
+      let newCmd = messages.pop().content.trim().concat(suffix);
+      bot.sendMessage(msg.channel, newCmd);
```

Comment:
Ah yes, I think I know the issue. I have another branch I'm going to merge in about 30 minutes that'll solve it.